frontend/packages/dev-console/integration-tests/features/Knative/knative-Create_event_sources.feature

```diff
@@ -1,40 +1,50 @@
 Feature: Create event sources
-    As a developer I want to create event sources for ApiServerSource, ContainerSource, CronJobSource, PingSource, SinkBinding. 
+    As a developer I want to create event sources 
 
 Background:
    Given open shift cluster is installed with Serverless and eventing operator
-   And open the project namespace "AUT-create-knative-event-source"
-   And user is on dev perspective
+   And user is at developer perspecitve
+   And user has selected namespace "aut-create-knative-event-source"
 
 
 @regression, @smoke
-Scenario: Different event source types display in event sources add page - Kn-07-TC03
-   Given user is on Add page 
+Scenario: Different event source enters display in event sources add page - Kn-07-TC03, Kn-08-TC02
+   Given user is at Add page 
    When user clicks on "Event Sources" card
-   Then user redirects to page with header name "Event Sources"
-   And able to see event source types like ApiServerSource, ContainerSource, CronJobSource, PingSource, SinkBinding
+   Then user will be redirected to page with header name "Event Sources"
+   And able to see event source enters like ApiServerSource, ContainerSource, CronJobSource, PingSource, SinkBinding
+
+
+@regression, @smoke
+Scenario: CamelSource event source - Kn-08-TC03
+   Given cluster is installed with knative Apache camel operator
+   And user is at developer perspecitve
+   And user is at Add page
+   When user clicks on "Event Sources" card
+   Then user will be redirected to page with header name "Event Sources"
+   And user able to see "Camel Source" event source type
 
 
 Scenario: knative eventing in operator backed - Kn-07-TC04
-   Given user is on Add page 
+   Given user is at Add page 
    When user clicks on "Operator Backed" card
-   Then user redirects to page with header name "Developer Catalog"
+   Then user will be redirected to page with header name "Developer Catalog"
    And able to see Knative Eventing card
 
 
 Scenario: Notifier message display in Event sources page when knative service is not available in namespace - Kn-10-TC01
-   Given user is on Add page
+   Given user is at Add page 
    But knative service is not available for selected namespace
    When user clicks on "Event Sources" card
-   Then user redirects to page with header name "Event Sources"
+   Then user will be redirected to page with header name "Event Sources"
    And able to see notifier with header "Event Sources Cannot be Created"
```

Comment:
```suggestion
   And user will be able to see notifier with header "Event Sources Cannot be Created"
```

Reply:
updated as 
 user is able to see notifier with header "Event Sources Cannot be Created"